api/src/main/java/ca/bc/gov/educ/api/penmatch/model/NicknamesEntity.java

```diff
@@ -33,6 +34,7 @@
    */
   @Id
   @Column(name = "NICKNAME2")
+  @Getter

```

Comment:
Hey Jinil, lombok data will generate getter for all fields , I believe this annotation is not required

Reply:
Thanks. I will remove it.